pkgs/sage-conf/pyproject.toml

```diff
@@ -1,3 +1,10 @@
 [build-system]
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "sage-conf"
+dynamic = ["version"]
+
+[tool.setuptools.dynamic]
+version = {file = ["VERSION.txt"]}
```

Comment:
It looks like you are trying to move things from setup.cfg to here, but now you'd need to remove it there to complete the job.

Reply:
Nope, just wanted to have the new conf package without setup.cfg. Removing the existing ones can be done in a follow-up.

Reply:
It's a regression in sage-conf to duplicate metadata.

Reply:
I don't understand enough about how setup cfg and pyproject toml interact to solve this. But it appears that the duplication of information results in no problem, so we (you?) can work on improving the setup.cfg file later on. It's definitely not in the scope of this PR to do this.

Reply:
I've now simply removed it from the setup.cfg file. Not sure if its working or something else is needed. Please test it and let me know.

Reply:
Thanks.

Reply:
So that's the first little step of #33577.

Reply:
I've cherry-picked this part into #36561 and completed the job there, please take a look.